### ISSUE TITLE
Direct editing enhancements for the desktop client

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#585]
When opening a direct editing link in the desktop we should either integrate the sidebar properly and hide the close button or just redirect to the files app to open the editor directly, if the user is currently logged in.

cc @misch7

**Comment by julien-nc** [comment]
@juliushaertl Is this still relevant nowadays?

**Comment by juliushaertl** [comment]
Yep, still relevant. The redirect could happen in a similar way like https://github.com/nextcloud/richdocuments/pull/2419/files#diff-f6848853e94bfe83f31f5a03e8e8c1d086eb07a4ba5c6646b2c5001e05a18da2R424-R455

In addition if on desktop we should just hide the close button and share icon from here (as then there is noone taking care of those post messages emitted):
https://github.com/nextcloud/text/blob/ba1bc2cb2eef697252fa98b89b8d8321beda91d4/src/views/DirectEditing.vue#L33

**Comment by max-nextcloud** [comment]
In order to make sure this https://github.com/nextcloud/text/labels/good%20first%20issue is easy to parse I tried to provide all information needed in the issue description above.

If some aspects of it are still unclear please ask so i can clarify further.

If you think this should be done differently please open a new issue describing your solution. Let's keep this issue focused on the proposed solution to provide clarity for whoever wants to tackle it.

**Comment by juliusknorr** [comment]
Updated with some more details
